index.js

```diff
@@ -1 +1,51 @@
-// Rachelle Pestanas & Brett Sullivan
\ No newline at end of file
+// Rachelle Pestanas & Brett Sullivan
+const redBlock = document.getElementsByClassName("block--red")[0];
```

Comment:
You could have also used querySelectorAll ```const rockets = document.querySelectorAll('.block');``` in order to target all elements on lines 2 - 7.